MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -309,6 +309,7 @@
 		AE87207E22CF97B900D7DAB7 /* InstructionsCardCollectionDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = AE87207D22CF97B900D7DAB7 /* InstructionsCardCollectionDelegate.swift */; };
 		AEC3AC9A2106703100A26F34 /* HighwayShield.swift in Sources */ = {isa = PBXBuildFile; fileRef = AEC3AC992106703100A26F34 /* HighwayShield.swift */; };
 		AED6285622CBE4CE00058A51 /* ViewController+GuidanceCards.swift in Sources */ = {isa = PBXBuildFile; fileRef = AED6285522CBE4CE00058A51 /* ViewController+GuidanceCards.swift */; };
+		B430D2FA25534FDC0088CC23 /* UserHaloCourseView.swift in Sources */ = {isa = PBXBuildFile; fileRef = B430D2F925534FDC0088CC23 /* UserHaloCourseView.swift */; };
```

Comment:
As I can see in https://github.com/mapbox/mapbox-navigation-ios/pull/2693 `Enable precise location to navigate` banner was added. Whenever we enable Precise Location banner disappears, but simulation banner is not present. @jill-cardamon, @1ec5 is it intentional? In my opinion it'd be a good idea to keep location simulation banner if Precise Location is switched on.

Reply:
That was not intentional. To be clear, you mean the `Simulating Navigation at 1x` banner correct? There's some tail work to do for #2693 (linked here: #2723), and I can look into making sure the simulation banner stays as part of that work.

Reply:
Yep, thanks for clarifying this. it'd be good if `Simulating Navigation at 1x` banner is available whenever Precise Location is On.